app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,29 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/wallpaper_main_background"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_main_activity"
+        style="@style/TitleMainActivityStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginTop="@dimen/margin_top_main_activity_text"
```

Comment:
I think that you can add this to the style

Reply:
Done!